app/src/app/client.cljs

```diff
@@ -227,18 +238,34 @@
 
 (defn page []
   [:div
-   [:div {:class "header"
-          :style {:display         "flex"
-                  :flex-direction  "row"
-                  :justify-content "space-around"}}
-    ;; layout hack: empty divs move the title and sign-off button more to the center
-    [:div]
-    [:div {:style {:font-size "1.3em"}} "ccDashboard"]
-    (if (domain/user-sign-in-state @domain/app-state)
-      [:a#logout {:href "/logout"}
-       [:i.icon-off.large-icon]]
-      [:div])
-    [:div]]
+   ;[:div {:class "header"
+   ;       :style {:display         "flex"
+   ;               :flex-direction  "row"
+   ;               :justify-content "space-around"}}
+   ; ;; layout hack: empty divs move the title and sign-off button more to the center
+   ; [:div]
+   ; [:a {:href  "/#"
+   ;      :style {:font-size "1.3em"}}
+   ;  "ccDashboard"]
+   ; (sign-in-only-content [:a#location {:href "/#location"
```

Comment:
currently the `Home` & `Locations` tabs are visible when logged out. Was this commented out to test something?